### PR TITLE
fix: Tap reporter uses correct index case

### PR DESCRIPTION
**Pull request by alimnios72**
The case index in a suite should be reset after executing the whole suite.

### CODE REVIEW

File: resources/roca_lib.brs [hunk 1]
```diff
@@ -179,7 +179,7 @@ sub __suite_exec(args as object)
     end for
     tap.exitSubTest()
 
-    index = args.index
+    index = 0
```
Comment (sjbarag) [inline resources/roca_lib.brs]: I'm torn about whether this should be `subTestIndex` or not!  If you get time, could you try a structure like this real fast?  I'm not sure if it'd produce a suite and a testcase both with index `1` in the output, and if that's okay per the `tap-parser` module!

```brightscript
describe("foo", sub()
    describe("bar", sub()
        it("sub-test 1", sub()
        end sub)

        it("sub-test 2", sub()
        end sub)
    end sub)

    it("test one", sub()
    end sub)
end sub)
```

Comment (alimnios72) [reply resources/roca_lib.brs]: Sure, gonna give it a try

Comment (alimnios72) [reply resources/roca_lib.brs]: Actually you were right, it didn't work on nested test cases but using `subTestIndex` does the trick

**Comment by sjbarag** [comment]
Published!  https://www.npmjs.com/package/@hulu/roca/v/0.4.1

**Pull request merged**